fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts

```diff
@@ -1476,13 +1513,35 @@ export class FwsCore {
 
   public readonly memos = new FwsMemos(this);
   public readonly normalChecklists = new FwsNormalChecklists(this);
-  public readonly abnormalSensed = new FwsAbnormalSensed(this);
   public readonly abnormalNonSensed = new FwsAbnormalNonSensed(this);
+  public readonly abnormalSensed = new FwsAbnormalSensed(this);
+  public ewdAbnormal: EwdAbnormalDict;
 
   constructor(
     public readonly fwsNumber: 1 | 2,
     public readonly bus: EventBus,
+    private readonly failuresConsumer: FailuresConsumer,
+    public readonly fws1Failed: Subscribable<boolean>,
+    public readonly fws2Failed: Subscribable<boolean>,
   ) {
+    this.ewdAbnormal = Object.assign(
+      {},
+      this.abnormalSensed.ewdAbnormalSensed,
+      this.abnormalNonSensed.ewdAbnormalNonSensed,
+    );
+
+    this.startupCompleted.sub((v) => {
+      if (v) {
+        this.turnaroundReset();
+      }
+    });
+
+    this.shutDownFor50MinutesCheckListReset.sub((v) => {
+      if (v) {
+        this.turnaroundReset();
```

Comment:
This might not achieve what you want as the confirmation node is set to false if someone manually resets a normal checklist, as such, it won't reset the abnormal procedures that way. Maybe those should reset directly on a phase 12 -> 1 changeover (phase12ShutdownMemoryNode) or they don't reset at all without a fws reset?